### ISSUE TITLE
Link underline should be the same as FG color 

### ISSUE THREAD

**Issue by Tyriar** [xtermjs/xterm.js#957]
Regression introduced in https://github.com/sourcelair/xterm.js/pull/938

![image](https://user-images.githubusercontent.com/2193314/30223657-9bc3e2d8-9480-11e7-9e40-52ae4091420c.png)

**Comment by peacefullatom** [comment]
I'd like to grab this :)

**Comment by Tyriar** [comment]
@peacefullatom great! 😃 

The link underlines are rendered from this file: https://github.com/xtermjs/xterm.js/blob/master/src/renderer/LinkRenderLayer.ts

You can check this code for how to get the foreground color:

https://github.com/xtermjs/xterm.js/blob/0e45909c7e79c83452493d2cd46d99c0a0bb585f/src/renderer/TextRenderLayer.ts#L139

**Comment by peacefullatom** [comment]
@Tyriar, thank you for hints, they were helpful!

This [comment](https://github.com/xtermjs/xterm.js/pull/938/files#r137778166) by @mofux was very informative as well.

If there is anything I can add/improve just tell me.

Cheers! :)
